apps/src/code-studio/pd/application_dashboard/form_data_edit.jsx

```diff
@@ -0,0 +1,171 @@
+/**
+   * Loads an application from the API by a supplied applicationId and passes it to
+ * the applicationData prop of the child component.
+ */
+
+import React, {PropTypes} from 'react';
+import {
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  ButtonToolbar,
+  Button,
+  Panel,
+  Table
+
+} from 'react-bootstrap';
+import parseJson from 'json-parse-better-errors';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  error: {
+    color: color.red
+  }
+};
+
+export default class FormDataEdit extends React.Component {
+  static propTypes = {
+    applicationId: PropTypes.string.isRequired,
+    applicationData: PropTypes.shape({
+      course_name: PropTypes.string,
+      application_type: PropTypes.oneOf(['Facilitator', 'Teacher']),
+      form_data: PropTypes.object.isRequired
+    }).isRequired,
+    onUpdate: PropTypes.func.isRequired
+  };
+
+  static contextTypes = {
+    router: PropTypes.object.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = this.getInitialState();
+  }
+
+  getInitialState() {
+    return {
+      formData: JSON.stringify(this.props.applicationData.form_data, null, 2),
+      parseError: null,
+      saveErrors: null
+    };
+  }
+
+  handleChange = event => {
+    // this.props.onUpdate(event.target.value);
```

Comment:
Yes, good catch! I'll remove the [prop](https://github.com/code-dot-org/code-dot-org/pull/20520/files#diff-27e14a057e593a326a447025f0172ac2R34) entirely since it's not being used.